### PR TITLE
Update for FreeBSD

### PR DESCRIPTION
**Pull request by KouichiHirabayashi**
In FreeBSD, <malloc.h> has been abolished and <stdlib.h> has been used.

### CODE REVIEW

File: src/femlib/CheckPtr.cpp [hunk 1]
```diff
@@ -27,6 +27,8 @@
  */
 #if __APPLE__
 #include <malloc/malloc.h>
+#elif __FreeBSD__
+#include <stdlib.h>
 #else
 #include <malloc.h>
 #endif
```
Comment (prj-) [inline src/femlib/CheckPtr.cpp]: Could you replace all this by `#include <cstdlib>` as suggested by @cdoucet in #118, please?

**Comment by KouichiHirabayashi** [comment]
> Could you replace all this by `#include <cstdlib>` as suggested by @cdoucet in #118, please?

I think that's better, but I can't test Mac, Linux, or Windows in my environment.

**Comment by prj-** [comment]
> Could you replace all this by `#include <cstdlib>` as suggested by @cdoucet in #118, please?
> I think that's better, but I can't test Mac, Linux, or Windows in my environment.

We will do the testing. It's a standard header, so it should work as intended.

File: Makefile.am [hunk 2]
```diff
@@ -278,8 +278,8 @@ install-exec-local:: bin/script/PostInstall.sh FreeFEM-documentation.pdf
 	cd idp; for i in *.idp; do \
 	 if [ -f $$i ] ; then 	$(INSTALL)  -m 555 $$i $(DESTDIR)$(ff_prefix_dir)/idp; fi; done
 
-bin/script/PostInstall.sh:./Makefile bin/script/PostInstall.m4
-	m4 bin/script/PostInstall.m4 "-DFF__FVER=$(PACKAGE_VERSION)" "-DFF_BINDIR=$(bindir)" "-DFF__DATADIR=$(pkgdatadir)" > bin/script/PostInstall.sh
+bin/script/PostInstall.sh:./Makefile bin/script/PostInstall.m4Makefile.am
```
Comment (prj-) [inline Makefile.am]: This change is wrong, remove `Makefile.am` at the end of the line, please.

**Comment by prj-** [comment]
Thank you for your contribution.

**Pull request merged**